docs/problem-statement.md

```diff
@@ -5,7 +5,7 @@
 - [Webhooks](#webhooks)
 	- [Problem](#problem)
 	- [Proposed solution](#proposed-solution)
-- [User accessing the Jenkins UI/API](#user-accessing-the-jenkins-uiapi)
```

Comment:
this is actually not related. This is not a reference to iapi as in the path of the Idler API. This is just a Markdown internal cross link. 

Since you also changes this in the anchor it will most likley work, but I recommend to drop this change.